acvm_js/src/execute.rs

```diff
@@ -23,6 +24,11 @@ impl SimulatedBackend {
     }
 }
 
+#[wasm_bindgen(js_name = "createBackend")]
+pub async fn create_backend() -> SimulatedBackend {
```

Comment:
We're not creating a backend here but a `BlackBoxFunctionSolver`. If we're exposing this externally then we should name these correctly.

```suggestion
pub async fn create_black_box_solver() -> WasmBlackBoxFunctionSolver {
```

Reply:
So we should basically rename `SimulatedBackend` itself right?

Reply:
Renamed `SimulatedBackend` -> `WasmBlackBoxFunctionSolver`